src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -70,6 +70,7 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     @Input() width: number;
     @Input() keyAction: KeyAction;
     @Input() keybindAnimationEnabled: boolean;
+    @Input() popoverEnabled: boolean;
```

Comment:
On a key level (component level) there should be no expectation about popover or any surrounding component, or about parents. 
If you are thinking in components, then a component should work in every use case. e.g. If I want to add a key somewhere e.g. on tutorial page where definitely will not be any popover used, then `popoverEnabled` would be weird property.